app/containers/Switch.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Switch as SwitchRN } from 'react-native';
```

Comment:
@SanjaySargam add an empty line. You should separate path imports with empty line

Reply:
<img width="1352" alt="Screenshot 2023-12-28 at 00 12 30" src="https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/61021881/845d5053-ea2c-4da5-9d4e-c4c1347d561f">